src/puppet.ts

```diff
@@ -689,6 +689,7 @@ export abstract class Puppet extends EventEmitter {
    * Friendship
    *
    */
+  public abstract async friendshipSearch (searchId: string) : Promise<void>
```

Comment:
Because we can not get the real wxid before we add friend, so the `contactId` which we returned is same as the input parameter, `phone` or `weixin`.

So I think it is no need to return `contactId` here.